### ISSUE TITLE
ECDSA support

### ISSUE THREAD

**Issue by kladkogex** [intel/cryptography-primitives#24]
Does ipp-crypto support the variant of ECDSA used by Bitcoin and Ethereum (the "k" curve)?

If not , how hard would it be to implement it?

We are using SGX at SKALE and need this variant of ECDSA.

If someone explains what file needs to be modified, we are happy to contribute ...

**Comment by skirillo** [comment]
No, ipp-crypto supports only EC over prime, but NOT over GF(2) extension

**Comment by skirillo** [comment]
No technical problem. It's mainly an organization problem. IPP Team needs strong case

**Comment by kladkogex** [comment]
Well ) If they want IPP used for blockchain pretty much all major blockchains use this )

**Comment by rcao8** [comment]
@kladkogex Is its feature request is still relevant to your project? We can enhance the functionality if there is strong request from community.
